kratos/python/add_variable_utils_to_python.cpp

```diff
@@ -462,6 +462,8 @@ void AddVariableUtilsToPython(pybind11::module &m)
         .def("SumElementVectorVariable", &VariableUtils::SumElementVectorVariable)
         .def("AddDof", &VariableUtils::AddDof<Variable<double>>)
         .def("AddDof", &VariableUtils::AddDofWithReaction<Variable<double>>)
+        .def_static("AddDofsList", &VariableUtils::AddDofsList)
+        .def_static("AddDofsList", &VariableUtils::AddDofsWithReactionsList)
```

Comment:
```suggestion
        .def_static("AddDofsWithReactionsList", &VariableUtils::AddDofsWithReactionsList)
```

Reply:
I followed the same criterion that our current `AddDofs` has.